server/app/views/applicant/TrustedIntermediaryDashboardView.java

```diff
@@ -78,7 +78,7 @@ public Content render(
                 h2(tiGroup.getDescription()).withClasses("ml-2"),
                 hr(),
                 renderHeader("Add Client"),
-                BaseHtmlView.requiredFieldsExplanationContent(),
+                requiredFieldsExplanationContent(),
```

Comment:
Given that `TrustedIntermediaryDashboardView` extends `BaseHtmlView`, it isn't necessary to specify `BaseHtmlView` here.